packages/expo-updates/ios/EXUpdates/Update/RawManifest/EXUpdatesRawManifest.h

```diff
@@ -0,0 +1,42 @@
+//  Copyright © 2021 650 Industries. All rights reserved.
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol EXUpdatesRawManifestBehavior <NSObject>
+
+# pragma mark - Raw JSON
+
+- (NSDictionary *)rawManifestJSON;
+
+# pragma mark - Field getters
+
+- (NSString *)rawID;
+- (NSString *)sdkVersion;
```

Comment:
nullable

Reply:
Yeah, this'll need a treatment similar to what we do on android, where we have both a `sdkVersionNullable` and a `sdkVersion`. Then depending on the callsite we'll choose one or the other. I plan to do this when I move the nullthrows checks to these method implementations though since the callsites are currently mixed.